sparts/tasks/dbus.py

```diff
@@ -24,6 +26,11 @@
 import glib
 import time
 
+# always init threads before calling any dbus code
+glib.threads_init()
```

Comment:
> Initializing stuff like this in module scope is problematic for a variety of reasons and I'd like to avoid it if possible.

Ok. I guess anyone using the old glib bindings would call threads_init() somewhere at the beginning of the program anyway, so it's fine to move these changes back to where they were originally.